### ISSUE TITLE
[High] Chrome + Jaws: Add button (+): Name, role & State are not defined for Action button (+)

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8479]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to any chat using Tab key and press Enter to select it.
3. Chat screen is appearing, Navigate to Action (+) button using Tab key and press using Enter to select it.
4. Observe that Name , role & State ate not defined for Action(+) button.

## Expected Result:
Name should be defined for + button so that screen reader will announce the button with its name. Screen reader should announce like add(+) button.
Name = "Action ", Role= "Button" & State = "Collapse/expanded".

## Actual Result:
Name, role & State are not defined for Action button (+).  Screen reader is reading some extra information or silent when focus lands on the Action(+) control.

**Other occurrences**
>Same issue repro on PR #8581, #8765, #8809, #9212

## Workaround:
Yes

## Area issue was found in:
Chat Editable text box.

## Failed WCAG checkpoints
4.1.2

## User impact:
How does this affect the user?

## Suggested resolution:
What can we do to fix the issue?

## Platform:
- Web
- iOS
- Desktop App

**Version Number:**  v1.1.49-1
**Reproducible in staging?:** Yes
**Issue reported by:** Stuti

Uploading 7660_Name role and state is not defined for + action button.mp4…

https://user-images.githubusercontent.com/97600950/161667274-eb3d96f2-adf4-4863-b184-730335e4981b.mp4
